### PR TITLE
Jenkins: reduce VM RAM size to 10 / 16 GB

### PR DESCRIPTION
**Pull request by maelk**
Ubuntu : from 16 to 10 GB
CentOS : from 32 to 16 GB

### CODE REVIEW

File: jenkins/scripts/integration_test.sh [hunk 1]
```diff
@@ -49,9 +49,9 @@ fi
 
 if [ "${IMAGE_OS}" == "Ubuntu" ]
 then
-  TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-4C-16GB-200GB}"
+  TEST_EXECUTER_FLAVOR="${TEST_EXECUTER_FLAVOR:-4C-8GB-200GB}"
```
Comment (fmuyassarov) [inline jenkins/scripts/integration_test.sh:52]: I'm wondering if we can reduce it to 16GB in line 47 as well

Comment (maelk) [reply jenkins/scripts/integration_test.sh:52]: I didn't want to touch something that I have not been working with. In theory, I would say yes, but I'll leave it up to you.

Comment (fmuyassarov) [reply jenkins/scripts/integration_test.sh:52]: Yes, sure. I'll check that.

**Comment by fmuyassarov** [comment]
Centos job is failing because of a bug in metal3-dev-env, not related to this PR. That issue should be fixed in [#350](https://github.com/metal3-io/metal3-dev-env/pull/350)

**Pull request merged**